app/src/main/java/com/isc/hermes/database/Hermesdb.java

```diff
@@ -0,0 +1,44 @@
+package com.isc.hermes.database;
+
+import com.mongodb.client.MongoDatabase;
+
+/**
+ * This class is responsible for generating the connection to the hermes database.
+ */
+public class Hermesdb {
+
+    private Connection connection;
+    private MongoDatabase database;
+    private static Hermesdb hermesdb;
+
+    /**
+     * It is a singleton method to have a single instance of the hermes database connection.
+     *
+     * @return only connection created.
+     */
+    public static Hermesdb getInstance() {
+        if (hermesdb == null)
+            hermesdb = new Hermesdb();
+
+        return hermesdb;
+    }
+
+    /**
+     * This is a constructor method to initialize their variables.
+     */
+    private Hermesdb() {
+        this.connection = Connection.getConnection();
+    }
+
+    /**
+     * This is a method returns the hermes database obtained.
+     *
+     * @return only hermes database connection.
+     */
+    public MongoDatabase getHermesDatabase() {
+        if (database == null)
+            database = connection.getDatabase().getDatabase("hermesmapdb");
```

Comment:
You could define the database name as a constant just like you did in your "Incidents" class.
Example: `private static final String DATABASE_NAME = "hermesmapbd"`

Reply:
done